### ISSUE TITLE
training information json

### ISSUE THREAD

**Issue by Lee-Gihun** [Lee-Gihun/Pytorch-OSI-setup#22]
For now, we only save the name of dataset.

But for the cases of user didn't set default valid size(5,000) we need to provide more information.

1) validation size
2) random seed for valid set
3) the transformation for train/valid/test
4) batch size of dataloader (and maybe some information for batch sampler etc)
